Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -270,7 +261,7 @@ install_package "ethtool"
 			}
 
 			# Verify the kernel panic, call trace or fatal error
-			$calltrace_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -iE '(call trace|fatal error)'" -ignoreLinuxExitCode:$true
+			$calltrace_filter = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "dmesg | grep -iE '(call trace|fatal error)'" -runAsSudo -ignoreLinuxExitCode:$true
 
```

Comment:
Do we need "-runAsSudo"? If the user is not root, we also can execute dmesg.

Reply:
That is not true all the times. Please see SLES case.

lisa@juhlee:~> dmesg
dmesg: read kernel buffer failed: Operation not permitted
lisa@juhlee:~> cat /etc/os-release
NAME="SLES"
VERSION="12-SP5"
VERSION_ID="12.5"
PRETTY_NAME="SUSE Linux Enterprise Server 12 SP5"
ID="sles"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:12:sp5"
lisa@juhlee:~>